### ISSUE TITLE
Crash when enabling 3D preview of multiplayer Character Models

### ISSUE THREAD

**Issue by EliteCombineSoldier** [w23/xash3d-fwgs#325]
Error Message:
![‏‏لقطة الشاشة (51)](https://user-images.githubusercontent.com/51729858/147733240-be4a74c5-f361-46bb-bb41-d42107307ef8.png)

 Actions to perform : Just go to the multiplayer Then Go to customize then enable 3d preview 

 - OS : Windows 10
 - GPU vendor and model : Nvidia GTX 1660 super
 - Driver version : 497.29

**Comment by 0x4E69676874466F78** [comment]
Да, после нажатия на галку "Show 3D Preview" падает, при этом оно запоминает это (в config.cfg `ui_showmodels "1"`) и после повторного запуска уже в это меню не пускает, сразу падает.

**Comment by w23** [comment]
That thing calls into rendering in a way that I haven't yet even looked at. At this moment vulkan renderer only supports regular gameplay rendering. Anything beyond that is very low priority for now.

We'll fix this eventually.
